src/dart-2.md

```diff
@@ -5,23 +5,41 @@ title: Dart 2 Migration
 description: How Dart 2 is different from Dart 1.x, and how you can convert your code to work with Dart 2.
 ---
 
-**This page is under construction.**
-It aims to help developers migrating from Dart 1.x to Dart 2.
+Here's the general process for migrating to Dart 2,
+from either Dart 1.x or an earlier version of Dart 2:
 
+1. Get an up-to-date version of the Flutter or Dart SDK.
+   * [Flutter SDK instructions][Flutter SDK install]
+   * [Dart SDK instructions][Dart SDK install] (VM or web)
+2. Upgrade the packages your app depends on.
+   * Flutter: [`flutter packages upgrade`][flutter package upgrade]
+   * Dart VM or web: [`pub upgrade`][pub upgrade]
+3. Run the [dart2_fix tool.][dart2_fix]
```

Comment:
nit: the period should be outside of the link

Reply:
and I see this in two links further down the page, so you may have a different opinion about where to put the periods :)

Reply:
Normally, I'd agree with you, but we have little icons for external links that look bad when they come before a period or comma.

Reply:
(I've been considering writing a plugin to handle that.)